lib/overcommit/hook/pre_push/git_lfs.rb

```diff
@@ -0,0 +1,21 @@
+module Overcommit::Hook::PrePush
+  # Invokes Git LFS command that uploads files tracked by Git LFS to the LFS storage
+  #
+  # @see https://git-lfs.github.com/
+  class GitLfs < Base
+    def run
+      result = execute(['command', '-v', 'git-lfs'])
```

Comment:
Is it always safe to assume that a non-zero exit code means `git-lfs` wasn't installed? We could run a `which git-lfs` first for better precision.